README.md

```diff
@@ -1,2 +1,80 @@
-# vertex-web-sdks
-Monorepo for the Vertex Front-End SDK codebase
+# Vertex Web Monorepo
+
+Welcome to the Vertex Web repo. This is a monorepo that contains our front-end
+libraries and sdks at Vertex.
+
+## Structure
+
+- `./packages`: The packages to be made available on NPM. This directory contains
+  subdirectories that represent different categories of packages:
+  - `./libs`: The libs directory contains libraries that are intended to be
+    reusable across other applications and/or libraries. These are normally
+    published to our private NPM repository.
+  - `./sdks`: The sdks directory contains NPM packages for Vertex's public facing
+    SDKs.
```

Comment:
Thinking that we keep the packages in this repo flat. I think we'll end up with around 10-12 packages which is manageable with a flat structure.